pkg/proxy/iptables/proxier_test.go

```diff
@@ -2426,4 +2425,53 @@ COMMIT
 	assert.NotEqual(t, expectedIPTablesWithSlice, fp.iptablesData.String())
 }
 
+func TestDetectLocalByCIDR(t *testing.T) {
+	cases := []struct {
+		cidr                    string
+		ipt                     utiliptables.Interface
+		jumpIfInput             []string
+		expectedJumpIfOutput    string
+		jumpIfNotInput          []string
+		expectedJumpIfNotOutput string
+		chain                   string
+	}{
+		{
+			cidr:                    "10.0.0.0/14",
+			ipt:                     iptablestest.NewFake(),
+			jumpIfInput:             []string{"-A", "KUBE-SVC-ABC", "-m", "comment", "--comment", `"Redirect pods trying to reach external loadbalancer VIP to clusterIP"`},
```

Comment:
All these cases can be stripped down.  The "-A KUBE-SVC-ABC" etc stuff is just noise, obscuring the test's intent

Reply:
so the idea was to get a visual feel for how the final rule would look like. So it's noise from a test perspective, the idea was to give the human reading it a sense of the kind of rule that would run.

Reply:
I'd prefer the test be easier to read :)